src/Infrastructure/BotSharp.Logger/Hooks/TranslationResponseHook.cs

```diff
@@ -0,0 +1,54 @@
+using BotSharp.Abstraction.Agents;
+using BotSharp.Abstraction.Infrastructures.Enums;
+using BotSharp.Abstraction.Translation;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BotSharp.Logger.Hooks
+{
+    public class TranslationResponseHook : ConversationHookBase
+    {
+        private readonly IServiceProvider _services;
+        private readonly IConversationStateService _states;
+        private const string LessenCopilot = "2cd4b805-7078-4405-87e9-2ec9aadf8a11";
```

Comment:
Can you use message.CurrentAgentId instead of the hardcode agent id?

Reply:
ok